test/syn_test.js

```diff
@@ -68,7 +68,8 @@ QUnit.test("scrollTop triggers scroll events", function () {
 	}, 13);
 });
 
-if (!syn.skipFocusTests) {
+//if (!syn.skipFocusTests) { // test is causing firefox to hang. TODO: Fix this!
```

Comment:
Is the test still causing Firefox to hang?

Reply:
Yes. I just noticed it when trying to do this check-in. It might be a syndrome of newer versions of firefox. Will need investigation.